CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to stream-chat-react
+
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Asking questions](#asking-questions)
+- [Filing an issue](#filing-an-issue)
+- [Contribution Standards](#coding-standards)
+- [Submission Guidelines](#submission-guidelines)
+- [Signing the CLA](#contributor-licence-agreement)
+
+
+## <a name="asking-questions"></a>Are you looking for answers?
+There are many ways you can get your questions answered. It can be hard to decide, where to begin if you are just starting out. We suggest you take a look at the resources in the following order
```

Comment:
```suggestion
There are many ways you can get your questions answered. It can be hard to decide, where to begin if you are just starting out. We suggest you take a look at the resources in the following order:
```

Reply:
Fixing typo